docs/_static/auth/iam.yaml

```diff
@@ -562,6 +562,290 @@ paths:
                 "$ref": "#/components/schemas/VerifiableCredential"
         default:
           $ref: '../common/error_response.yaml'
+  /iam/{did}/start-oid4vci-issuance:
+    post:
+      operationId: startOid4vciIssuance
+      summary: Start the Oid4VCI authorization flow.
+      description: |
+        Initiates an Oid4VCI flow to request an VC from a Credential Issuer.
+
+        error returns:
+        * 400 - one of the parameters has the wrong format or an OAuth error occurred
+        * 412 - the organization wallet does not contain the correct credentials
+        * 503 - the authorizer could not be reached or returned an error
+      tags:
+        - auth
+      parameters:
+        - name: did
+          in: path
+          required: true
+          description: The DID of the requester, a Wallet owner at this node.
+          schema:
```

Comment:
use content: plain/text:schema: due to oapi codegen bug.

Reply:
Got it, changed.